src/libs/actions/Report.js

```diff
@@ -1049,21 +1049,35 @@ function deleteReportComment(reportID, reportAction) {
  * @param {Number} reportID
  */
 function openReport(reportID) {
-    const sequenceNumber = getMaxSequenceNumber(reportID);
-    API.write('OpenReport',
+    API.read('OpenReport',
```

Comment:
This feels like a `write()` to me as we are setting the last read sequence number in the command and will want to make sure that happens when we come back from offline.

Reply:
you are right it should be a write

Reply:
👍

Reply:
Please make this a `write()`